helloworld-springboot/README.md

```diff
@@ -64,6 +64,16 @@ gcloud components update app-engine-java
 gcloud components update
 ```
 
+### Configure the `app.yaml` descriptor
+
+The [`app.yaml`][app-yaml] descriptor is optional, and is used to describe
+URL dispatch and resource requirements.  We recommend that you initially
+configure [`manual_scaling`][manual-scaling] and disable 
+[health-checks][health-checking].
```

Comment:
We should just put that in the app.yaml.  You can say somewhere that this sample has health checks disabled and is set to manual scaling = 1, you will probably wish to change this for a production environment.